mitmproxy/tools/console/consoleaddons.py

```diff
@@ -11,6 +11,29 @@
 from mitmproxy.tools.console import signals
 from mitmproxy.tools.console import keymap
 
+# We redefine these here for now to avoid importing Urwid-related guff on
+# platforms that don't support it, and circular imports. We can do better using
+# a lazy checker down the track.
```

Comment:
I searched through the project and did not find any duplicated definition... So no more **redefinition** here and I guess this comment should be removed.